lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts

```diff
@@ -219,8 +221,8 @@ export class StartTaskCloudComponent implements OnInit, OnDestroy {
 
     public whitespaceValidator(control: UntypedFormControl) {
         const isWhitespace = (control.value || '').trim().length === 0;
-        const isValid = control.value.length === 0 || !isWhitespace;
-        return isValid ? null : { whitespace: true };
+        const isValidate = control.value.length === 0 || !isWhitespace;
```

Comment:
Why did you just change the name of the variable here?

Reply:
I have imported & used isValid from date-fns which is giving an error here so, I have changed the variable name.

Reply:
Okay in this case can you please name it like `isControlValid`? `isValidate` sounds a bit weird to me

Reply:
I have updated the variable name.